Python/Product/Debugger.Concord/ModuleManager.cs

```diff
@@ -160,6 +165,27 @@ public static void PyCode_New(DkmThread thread, ulong frameBase, ulong vframe, u
                 }.SendLower(process);
             }
 
+            public static void _PyCode_New(DkmThread thread, ulong frameBase, ulong vframe, ulong returnAddress) {
```

Comment:
Does this function name make sense if you know what's going on here?

Reply:
Yeah it's the name of a function in the python dll. It has to match the name.